app/controllers/events_controller.rb

```diff
@@ -0,0 +1,52 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: %i[show edit update attend]
+  # GET /events or /events.json
+  def index
+    @events = Event.all
+  end
```

Comment:
As requested earlier on, group the events controller index action into past and upcoming events. Follow the previous reviewer's example to get a headstart on this feature.